src/utils/queryATLAS.ts

```diff
@@ -0,0 +1,80 @@
+import { ApolloClient, InMemoryCache, gql } from "@apollo/client";
+import { AcaiRecord } from "../../types";
+
+const client = new ApolloClient({
+  uri: "https://acai-resources-preview---symphony-api-svc-prod-25c5xl4maa-uk.a.run.app/graphql/",
+  cache: new InMemoryCache(),
+});
+
+export async function sendQuery(usfmRef: string): Promise<AcaiRecord[]> {
+  try {
+    console.log(`Querying ATLAS for USFM reference: ${usfmRef}`);
+
+    const query = gql`
+      query ACAIRecordsInPassage($acaiRecordsFilters: AcaiRecordFilter) {
+        acaiRecords(filters: $acaiRecordsFilters) {
+          id
+          label
+          description
+          recordType
+          uri
+        }
+      }
+    `;
+
+    const { data } = await client.query({
+      query,
+      variables: {
+        acaiRecordsFilters: {
+          scriptureReference: {
+            usfmRef,
+          },
+        },
+      },
+    });
+
+    if (data.acaiRecords && data.acaiRecords.length > 0) {
+      return data.acaiRecords;
+    } else {
+      throw new Error("No ACAI records found");
+    }
+  } catch (error) {
+    console.error("Error querying ATLAS:", error);
+    throw error;
+  }
+}
+
+function formatVerseRef(bookId: string, verseRef: string): string {
+  const parts = verseRef.split("-");
+
+  if (parts.length === 1) {
+    // Single verse or chapter:verse
+    return `${bookId} ${verseRef}`;
```

Comment:
@sampjvv A small easy fix I noticed when testing out the branch in the VS Code debugger...doing a "book-level" query didn't work.

The change below would support book-level queries, e.g. `PHM`

```suggestion
    return `${bookId} ${verseRef}`.trim();
```
<img width="1290" alt="image" src="https://github.com/user-attachments/assets/8b197c65-17bc-4d15-aa8d-2b132a35ad24">

Reply:
got it, will do!